mne/annotations.py

```diff
@@ -41,7 +41,7 @@ class Annotations(object):
     ----------
     onset : array of float, shape (n_annotations,)
         Annotation time onsets relative to the ``orig_time``, the starting time
-        of annotation acquisition.
+        of annotation acquisition (in seconds).
```

Comment:
written like this is not clear if the `starting time of annotation acquisition` is in seconds or if the `onsets` should be in seconds. 
I recommend to change the entire comment instead of just adding the unit.